app/views/admin/deploy_groups/show.html.erb

```diff
@@ -61,6 +61,11 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
+        <form style="display: inline-block" method="post" action="<%= deploy_all_admin_deploy_group_path(@deploy_group) %>">
+          <%= hidden_field_tag :authenticity_token, form_authenticity_token %>
+          <input type="hidden" name="missing_only" value="true">
+          <input type="submit" class="btn btn-default" value="Deploy Missing Projects" title="Deploys will be started for any stages that haven't yet been deployed or have failed all their deploys." >
+        </form>
```

Comment:
`data-params` and `data-method=post` doesn't work without `data-remote=true`. And we don't want remote=true for this link. So I had to recreate the form manually to get a submit button that would include the extra parameter in POST.